sandbox_scripts/QualiEnvironmentUtils/Sandbox.py

```diff
@@ -20,9 +24,10 @@ def __init__(self, reservation_id, logger):
             """:type : logging.Logger"""
             self.api_session = helpers.get_api_session()
             self.id = reservation_id
-
+            self.owner = helpers.get_reservation_context_details().owner_user
```

Comment:
also, now that we have the environmentPath, i would save it as well (self.environmentPath =..)